acquire/tools/decrypter.py

```diff
@@ -444,6 +461,22 @@ def main():
                     executor.shutdown(wait=True, cancel_futures=True)
                     break
 
+            if len(files) > 1:
+                results_string = textwrap.indent(
+                    "\n".join(
+                        f"{file} - {'Success' if success else 'Failed'}: {message}"
+                        for file, (success, message) in sorted(results.items())
+                    ),
+                    " • ",
+                )
+                (progress.console.log if progress else log.info)(
+                    f"\nDecrypt results (file - result: message):\n{results_string}"
+                )
+
+            if not all(results.values()) or tasks:
+                exit_code = 2 if any(results.values()) else 1
```

Comment:
You probably only want to check on the 1st item in `results.values()` otherwise the `any(...)` will always return `True`.

Reply:
True! Oversight after refactor :)